pkg/performanceprofile/profilecreator/profilecreatorfuncs_test.go

```diff
@@ -0,0 +1,166 @@
+package profilecreator
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/jaypipes/ghw/pkg/cpu"
+	"github.com/jaypipes/ghw/pkg/topology"
+)
+
+func TestSortTopology(t *testing.T) {
```

Comment:
why do we need to extract the `sortTopology` helper (and test a private function :\ )? Can we test the public `SortedTopology()` public function?

Reply:
followup PR: https://github.com/openshift/cluster-node-tuning-operator/pull/1217